services/src/main/java/org/exoplatform/news/NewsServiceImpl.java

```diff
@@ -1082,7 +1082,7 @@ public boolean canEditNews(String posterId, String spaceId) {
     return authenticatedUser.equals(posterId) || spaceService.isSuperManager(authenticatedUser)
         || currentIdentity.isMemberOf(PLATFORM_WEB_CONTRIBUTORS_GROUP, PUBLISHER_MEMBERSHIP_NAME)
         || currentIdentity.isMemberOf(currentSpace.getGroupId(), MANAGER_MEMBERSHIP_NAME)
-        || currentIdentity.isMemberOf(PLATFORM_ADMINISTRATORS_GROUP, "*");
+        || (currentIdentity.isMemberOf(PLATFORM_ADMINISTRATORS_GROUP, "*") && spaceService.isManager(currentSpace, currentIdentity.getUserId()));
```

Comment:
```suggestion
 
```
We need just to remove currentIdentity.isMemberOf(PLATFORM_ADMINISTRATORS_GROUP, "*"); because spaceService.isManager(currentSpace, currentIdentity.getUserId())); is already in the above condition